pyttman/core/internals.py

```diff
@@ -48,7 +50,8 @@ class Settings:
     aren't valid settings.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, dictionary={}, **kwargs):
```

Comment:
This will cause a memory leak, where `dictionary={}` is used over and over again for every call. 
To fix this, use the default argument `None` as:

```py
def foo(data=None):
   if data is None:
      data = {}  # Created in the function scope, fixes the issue
```